File: go/orm/sample/migrations/V0002.go
package migrations

import (
	"github.com/kosmassp/SimpleOrm/go/orm"
	"github.com/kosmassp/SimpleOrm/go/orm/sample/migrations/Table/User"
)

type V0002 struct{}

func (V0002) Compose(version *orm.VersionBuilder) {
	version.Apply(user.V0002_AddDisplayName{})
}

File: php/samples/SimpleOrm.Sample/src/Migrations/V0002.php
<?php

declare(strict_types=1);

namespace SimpleOrm\Sample\Migrations;

use SimpleOrm\Migrations\MigrationVersion;
use SimpleOrm\Migrations\VersionBuilder;
use SimpleOrm\Sample\Migrations\Table\User\V0002_AddDisplayName;

final class V0002 extends MigrationVersion
{
    public function compose(VersionBuilder $version): void
    {
        $version
            ->apply(V0002_AddDisplayName::class);
    }
}
